cypress/e2e/specialFeatures/liteSiteWeight/index.cy.js

```diff
@@ -0,0 +1,93 @@
+import { assertPageWeight } from './assertions/liteSiteWeight';
+
+const liteTestSuites = [
+  {
+    path: '/hindi.lite',
+    service: 'hindi',
+    runforEnv: 'local',
```

Comment:
Can we include a page type in here, like we did for the other lite site test, and use it in the test description?